corehq/apps/app_manager/suite_xml/sections/resources.py

```diff
@@ -9,34 +9,21 @@ class FormResourceContributor(SectionContributor):
     section_name = 'xform_resources'
```

Comment:
I meant the code that processes a user registration submission and makes a user out of it. I think the entry point is [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/users/signals.py#L44)